### DISCUSSION TITLE
エゴアイテム強化の要望

### DISCUSSION THREAD

**Discussion by Aquashrimp** [hengband/hengband#216] [Ideas]
兜の赤外線視力エゴや冠の急速回復エゴなど、価値が低いエゴアイテムの強化をして頂きたいです
特定の指輪/アミュ限定エゴ(テレパシーアミュ専用の感知エゴなど)も中々見つけられるものではないので、レア度にふさわしい強化をしてもらってもいいかもしれません

**Comment by deskull-m** [comment]
弱いエゴのままにするなら拾える確率そのものを高めてそこそこの金策にできたり、強化するならそれはそれでよしという所でしょうか。念頭に入れます。

**Comment by gold-onion** [comment]
混沌武器の使い勝手が悪すぎてあまり使われていないようなので修正案です。

・テレポートアウェイで敵が飛んで行ってしまう問題
　　飛ぶ距離を遠近の2つに分けて遠距離は低確率にする
・変身するとHPが満タンになる問題
　　カメレオンみたいにHPは割合に変更　全部だと強すぎる場合は確率で振り分け
・殴る度にたまにランダムで5元素のいずれかのスレイが出る
　　フレーバーとしてなので威力は目指さない　1/5～1/10ぐらいの確率

**Reply by deskull-m** [reply]
そうですね、仰る通りにデメリットを緩和するか。メリット面を強化して、呪いエゴの扱いにするというのも昨今の調整としてはありかもしれません。

**Comment by RRyukkuriRyu** [comment]
クロークと光源にはどちらにも灼熱エゴが存在しますが、光源の方は火炎耐性がないので実質下位互換になってしまっています。
光源の方の灼熱エゴにも火炎耐性を付与してもいいのではないでしょうか。

**Reply by iks3** [reply]
つけました。

**Comment by daradarach** [comment]
新装備フラグ - 魔法防御上昇 #198 が実装の暁には防衛者エゴに魔法防御を実装しても、なんて話がありました

**Comment by daradarach** [comment]
|沌　で起こる各種効果を単体で発揮する効果の実装によって★武器にもうちょっと特徴を付けられたりしないかなと妄想しています。
- 既存
-- 打撃に吸血属性が付与される
-- 打撃に地震属性が付与される
- 単体実装無し
-- 打撃に混乱属性が付与される
-- モンスターをテレポートさせる
-- モンスターを変身させる

**Comment by ghost** [comment]
### 地震エゴのランダム発動抑止
強化とは異なるのですが、上のほうで出ている「混沌武器が使いにくい」を見ていてふと思いつきましたので。
地震エゴは通常攻撃時にランダムで地震が発生しますが、これは「.」で抑止できなかったように記憶しています。
地震は敵にダメージを与える一方でアイテムが消えたりといったデメリットもあり個人的にほとんど使いこなせていません。
これが抑止可能になれば、
　・低階層で使えるかも
　　地震エゴ1本あればシャベル不要でザックが節約できるというメリットがいちおうあるかと思います。
　・追攻つき地震エゴの活躍の機会が増える
　　+2までは見かけたと思います。追攻エゴの互換になるポテンシャルはあるかと思います。
という感じで多少日の目を見るかも……という気持ちがあります。

☆などで複数効果のうちの一つとして地震が付いた場合などどう抑止するのか……とかはちょっと悩ましい気もしますが、ご検討いただけますと幸いです。

**Reply by daradarach** [reply]
トランプエゴのランダムテレポートは呪われていない限りキャンセル可能なので、常にキャンセルするのが銘.なのであって、能動的にキャンセルできない地震を制御するのはだいぶ話が違いません？

**Reply by ghost** [reply]
> トランプエゴのランダムテレポートは呪われていない限りキャンセル可能なので、常にキャンセルするのが銘.

ご指摘ありがとうございます、その辺わかってなかったです。
「.」は「エゴの能力抑制」ではなく「テレポートしますか(y/n)に常に'n'回答する」みたいなUI側動作を制御するものなんですね。
別の強化の仕方がないか考えてみます。

**Reply by ghost** [reply]
ランダムで地震が発生する特性についてはその後よいアイデアを思いつかなかったので発案者としましては取り下げという事でお願いいたします。

**Reply by deskull-m** [reply]
了解です、その内巡回でなにか別案思いついたらまた考えるくらいにしておきましょう。

**Comment by ghost** [comment]
### アーチャーギルドで属性／スレイ付与
エゴ弓誕生でエゴ矢の存在感がさらに空気になりそうな気がしましたので何かテコ入れできないかと考えました。
現状アーチャーギルドでLv/5の修正値まで弓・矢を強化ができますが、矢玉にエゴ付与できるよう拡張するのはどうでしょうか？
個人的に矢玉を自作できない職業にとってエゴ矢は「まとまった数が揃えられない」→「ザックのスロット消費してまで持ち歩く利点がない」という感じで序盤なら金策に拾うぐらいで中盤以降は無視する状況です。
補充が容易になれば出番が増えるのではと思いました。
スレイ付与はバランス的に難しいかもしれませんが武器で/邪デ死、矢で/動をカバーなど装備の組合せで遊ぶ余地が増えて楽しいかも。

**Comment by ghost** [comment]
### 祝福エゴの強化
立て続けで申し訳ありません。空気過ぎて忘れていた祝福エゴについてもテコ入れできないでしょうか？
プリーストでもペナルティなしに刃物を装備できるエゴという立ち位置のようですが、プリーストでプレーしていても耐性・スレイが優先される場合が多く、上位互換とでもいうべき滅邪エゴ、聖戦者エゴもあることから序盤で引いた時ぐらいしか使いどころがない弱エゴの感があります。
そこで以下のような強化はどうでしょうか？
・「/デ」を付与
　悪魔祓いに効果がありそうなイメージなので。
・賢さ維持を付与
　神のご加護的な。
・ランダムな能力を1→2
　ランダムな能力が１つだとイマイチな能力が付いたり他装備と被ったり残念なことが多いので２つに強化。
　ここでうまく浮遊+αが付けばVer3新種族のマーフォークで使い道のあるエゴという立ち位置になるかも？
以上ご検討いただければ幸いです。

**Reply by iks3** [reply]
α17あたりで強化済みです。{+賢/死デ~善;祝} 光源+2、追加能力、追加上位耐性
これにα19から1/4で呪力耐性追加予定です。

**Comment by gitgita6** [comment]
**光源エゴの強化＆新案**
現状だと最終装備の光源枠は不朽の眼のと忍者の暗闇以外はほぼアーティファクト確定ですし
序盤中盤の選択肢も増えると楽しいと思います。
ちょっとヤバげなのも含めアイデアだけ列挙しました。

光明の　範囲+1　を範囲+1～5　まで強化（生成階層に応じ？）

雷光の　電撃耐性,電撃オーラ
灯台の　発動:太陽光線　自然領域のと同効果　地味なので元素エゴ武器の発動くらいには連続使用可能に
星光の　発動:スターライト　同杖のような連続使用はできない匙加減で

以下は幽鬼クロークとかみたく相当低い出現率にする（かアーティファクトの方がいいかも）
火山の　発動：悪魔領域の溶岩流（スカムに悪用されそうなのでクエスト内では発動不可？）火炎で傷付かない
光壁の　発動：短期間の反射
耐光の　耐閃光
耐暗の　耐暗黒
聖光の　破邪領域の聖なるオーラ（発動方式にするかどうかは出現確率と相談）

**Reply by deskull-m** [reply]
適応バージョンはともかくIssueの俎上には上げるべきと判断しました。

**Comment by ghost** [comment]
### 守りの指輪への呪力耐性付
守りの指輪は序盤や中盤の上位耐性が必須とまで言わない階層で他に有用な指輪が無い時に消極的に付けるぐらいであまり活かせてないので呪力耐性を付けて強化するのはどうでしょうか？
魔道具適正のある種族なら守りの指輪は火炎など属性防御＋ACボーナス系指輪に置き換わっていくと思いますが、魔道具下手で発動が活かせない(≒魔法防御も低い）種族に呪力耐性付の守りの指輪はそこそこ使い道があるかもと考えております。

案１：エゴなしでも付与
　守りエゴの盾・クロークに1/3で呪力耐性が付くので、守りの指輪にも1/3程度の確率で呪力耐性を付ける

案２：大防御エゴに付与
　守りの指輪は出やすいので呪力耐性付与がゲームバランス的に難しい場合。
　大防御エゴの守りの指輪に確定or高確率で呪力耐性を付ける。
　(とはいえ大防御エゴあまり見かけないのでこれはこれで拾った頃には追攻優先で使われない？)

案３：新規に守りの指輪に「魔防」エゴ(仮称)を設立
　１，２ともバランス的に難しい場合、守りの指輪用にエゴを新設し、確定で呪力耐性を付け、いいかんじの確率で出るようにする。
　（魔除けという意味だと指輪よりアミュレットにつけるエゴのような気もしますが）

以上、ご検討いただければ幸いです。

**Reply by deskull-m** [reply]
Issueに持っていきました。先のバージョンで議論したいと思います。

**Comment by deskull-m** [comment]
結構放置しっぱなしの案が残っているようなので折見てさらいます。ひとまずは手近なものだけでも。

**Comment by ghost** [comment]
### 魔力支配エゴの強化案
個人的にはビルドで*破壊*の杖を振れないようなキャラを作らないというのが一般的なプレースタイルなのか、あまり活用できていない気がします。
あえて魔道具下手なキャラでチャレンジする場合以外にも、*破壊*が不要な中層あたりで使う機会を増やすため、以下のような強化はいかがでしょうか？

１：呪力耐性付与
　適当な割合で呪力耐性を付与。

２：発動で魔力防御を一定ターン上昇
　現行で魔力防御を向上する発動が実装済かは分かりませんが。

すでにVer3開発も佳境のようですので、それ以後のネタとしてでもご検討いただければ幸いです。

**Reply by dis-** [reply]
元々有用な物品である上に、杖・魔法棒が充填魔力吸収打撃で吸われにくくなる効果まで追加されているのでこれ以上は過剰強化と思われます。

**Reply by ghost** [reply]
ご確認ありがとうございます。
強化具合を見ようとしばしプレーしてみましたが、思うようなときに思うようなエゴアイテムが落ちてくるはずもなく。
何にせよすでにテコ入れ済でしたら仰る通り再強化は必要ないと思います。お騒がせいたしました。
